Library/Homebrew/cask/lib/hbc/quarantine.rb

```diff
@@ -12,10 +12,32 @@ def swift
       @swift ||= DevelopmentTools.locate("swift")
     end
 
+    # @private
```

Comment:
No need to mark this as private, since everything except the explicitly documented parts of `Formula` should be considered private.